src/examples/gallery.js

```diff
@@ -0,0 +1,24 @@
+/** @jsx jsx */
+import { jsx, Box, Flex } from "../code";
+
+export default () => (
+  <Flex sx={{ flexDirection: "column", minHeight: "100vh" }}>
+    <Box sx={{
+      display: 'grid',
+      flex: 1,
+      minHeight: '100vh',
+      gridTemplateAreas:
+        `"long-box long-box box-left box-right"` + ` ` +
+        `"long-box long-box wide-box wide-box"`
+      ,
+      gridTemplateColumns: 'repeat(4, 1fr)',
```

Comment:
Thanks for putting this together! I was originally trying to make sure all the layouts work for responsive layouts – is that something you think you could look into for this layout?

Reply:
Done 😎 Sorry for that, I didn't know that the order in an array is for handling media queries 😬